src/pool/srv_iv.c

```diff
@@ -1419,9 +1419,14 @@ int
 ds_pool_iv_srv_hdl_invalidate(struct ds_pool *pool)
 {
 	struct ds_iv_key	key = { 0 };
+	struct pool_iv_key	*pool_key;
```

Comment:
I don't understand this change, you create a variable on the stack but then don't reference it anywhere?  The only function called is d_hlc_get() so I'm not sure how this PR changes anything?

Reply:
it will initialize the pool_key to reference the key (key.key_buf), then initialize it, otherwise it will might cause Assertion in pool_iv_ent_alloc, which require the pin_entry_size > 0.

Reply:
But pool_key isn't used anywhere?

Reply:
1426 L?

Reply:
Ah I see it now.  I misread that as being the same as the lines below it and assigning a struct entry, I didn't realise it was setting a pointer.